ansible/roles/dashmate/tasks/build.yml

```diff
@@ -92,6 +92,16 @@
   when: git.changed # noqa: no-handler
   changed_when: true
 
+- name: Check if default config is set
```

Comment:
Why do we need this?

Reply:
Because immediately after, we run `dashmate config set`. This command will fail on the initial run of deploy tool, because the default config is not yet set.

Reply:
You mean config is not rendered yet?

Reply:
I mean `defaultConfigName` is not yet set, so commands like `dashmate config set ...` cannot work